src/lib/auth.js

```diff
@@ -0,0 +1,8 @@
+export default function postTokenToServer(tok) {
+	fetch('http://localhost:4000/login/google', {
```

Comment:
I'd like to set up a configuration for this, but this works.

Reply:
Yeah, a config file in the client will be good.